setup.py

```diff
@@ -18,6 +18,7 @@
   "prometheus-client==0.20.0",
   "protobuf==5.27.1",
   "psutil==6.0.0",
+  "pyamdgpuinfo==2.1.6",
```

Comment:
the problem is this fails when running on Mac.
we will need a special case for this.